master/getting-started/kubernetes/installation/gce.md

```diff
@@ -2,49 +2,21 @@
 title: Deploying Calico and Kubernetes on GCE
 ---
 
-These instructions allow you to set up a Kubernetes cluster with Calico networking on GCE using the [Calico CNI plugin][cni-plugin]. This guide does not setup TLS between Kubernetes components or on the Kubernetes API.
+There are several possibilities for [deploying Kubernetes on GCE][running-GCE-on-K8s]
+and with minimal adjustments Calico can be [installed](.) too.
```

Comment:
Maybe the link isn't good because we don't really want to 'endorse' one way over another.